### ISSUE TITLE
Double-colons in output even with --print-f77-parameter

### ISSUE THREAD

**Issue by jcburley** [CodethinkLabs/ofc#17]
The double-colons are unncessary and will prevent older tools from reading the resulting output (mainly a problem for `.inc` files). E.g.:

```
 $ cat decl.inc
      INTEGER FOO
      PARAMETER (FOO = 2)
$ ofc --print-f77-parameter --sema-tree decl.inc > decl-new-2.inc 
$ cat decl-new-2.inc
        INTEGER :: FOO
        PARAMETER (FOO = 2)
$
```

**Comment by jcburley** [comment]
Note that this is of concern if single `.inc` files are to be used by a codebase undergoing gradual conversion, which `ofc` does seem to support in some ways (such as by creating `.inc` files that can be INCLUDEd by both old and converted code due to being valid fixed-form as well as free-form files).

**Comment by ben-brewer-codethink** [comment]
For now I've changed to only print double colons when required.

Fixed in commit: e5950f1f7cff79fa391773e6b5a4b21a9e6efd9a
